deb-package/pkg/opt/dedis/dvoting/config/config.env

```diff
@@ -1,4 +1,8 @@
-# dela config
+# D-Voting global configuration.
+#
+# This file should not be updated unless for very specific use cases.
+# For user configuration see /etc/dedis/dvoting
+#
 export dela_bin="/opt/dedis/dvoting/bin/memcoin"
 export dela_data="/var/opt/dedis/dvoting/data/dela"
-export dela_proxy_pk=3c07e93b9d99032366f7d92697f8dc1337bf8bb617b3faa6389e94d18e3d0e40
+#
```

Comment:
`#` left over ?

Reply:
that was intentional but yeah, can be removed ^^